user-guide/queries/query-url.md

```diff
@@ -0,0 +1,16 @@
+# Querying URLs
+
+Using a URL based data source requires that the URL return the [_results JSON format_](https://redash.io/help-onpremise/how-rd-works/data-source-results-format.md)
+
+The query itself inside Redash will simply contain the URL to be executed (i.e. http://myserver/path/myquery) and result with all the data it has - similar to `select * from db`.
+
+Valid types of columns returned in results:
+TYPE_INTEGER = 'integer'
+TYPE_FLOAT = 'float'
+TYPE_BOOLEAN = 'boolean'
+TYPE_STRING = 'string'
+TYPE_DATETIME = 'datetime'
+TYPE_DATE = 'date'
```

Comment:
Let's move this list to the query results JSON format page & just list the types as:

* integer
* float
* ...